Samsung/LPVS
# Add Dashboard Page.

## Description

LPVS Front-End `Dashboard` Page.
This page provides an analysis of the created PR in the form of a Dashboard.

There are no more page upload plans.
After merge this PR the next PR relates to enhancement to the History page. #221 

## Type of change


# How Has This Been Tested?
1. Update environment variables of `docker-compose.yml`, `src/main/resources/application.properties`, `frontend/.env`  file.
2. Build the lpvs container : `docker compose build`
3. Run containers: `docker compose up`
4. See the lpvs home page with any browser: `http://localhost:7896`

The values required to implement the login function can be obtained from the link below.
Naver: https://developers.naver.com/products/login/api/api.md
Kakao: https://developers.kakao.com/docs/latest/en/kakaologin/common
Google: https://cloud.google.com/identity-platform/docs/web/google

![1](https://github.com/Samsung/LPVS/assets/57388014/4f44cad3-90a0-44a9-b898-3f36b55e43ea)
![2](https://github.com/Samsung/LPVS/assets/57388014/005e0496-e816-4f6a-81a1-eeb4cc049c5e)

**Test Configuration**:
* Java: v11
* LPVS Release: v1.1.0

# Checklist:
